cmd/tle/commands/commands_test.go

```diff
@@ -15,15 +18,65 @@ func Test_ParseDuration(t *testing.T) {
 	}
 
 	tests := []test{
-		{name: "parseDay", duration: "1d", date: time.Now(), expected: 24 * time.Hour, err: nil},
-		{name: "parseMonth", duration: "1M", date: time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC), expected: time.Duration(31*24) * time.Hour, err: nil},
-		{name: "parseYear", duration: "1y", date: time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC), expected: time.Duration(365*24) * time.Hour, err: nil},
-		{name: "parseInvalid", duration: "1C", date: time.Now(), expected: time.Second, err: ErrInvalidDuration},
+		{
+			name:     "parseDay",
+			duration: "1d",
+			date:     time.Now(),
+			expected: 24 * time.Hour,
+			err:      nil,
+		},
+		{
+			name:     "parseMonth",
+			duration: "1M",
+			date:     time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC),
+			expected: time.Duration(31*24) * time.Hour,
+			err:      nil,
+		},
+		{
+			name:     "parseYear",
+			duration: "1y",
+			date:     time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC),
+			expected: time.Duration(365*24) * time.Hour,
+			err:      nil,
+		},
+		{
+			name:     "parseMixed",
+			duration: "1y1M1s",
+			date:     time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC),
+			expected: time.Duration(365*24)*time.Hour + time.Duration(31*24)*time.Hour + time.Duration(1)*time.Second,
+			err:      nil,
+		},
+		{
+			name:     "parseMixedInAFunnyOrder",
+			duration: "2s1y1M",
+			date:     time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC),
+			expected: time.Duration(365*24)*time.Hour + time.Duration(31*24)*time.Hour + time.Duration(2)*time.Second,
+			err:      nil,
+		},
+		{
+			name:     "multiDigitParsesCorrectly",
+			duration: "203m",
+			date:     time.Date(2022, 01, 01, 0, 0, 0, 0, time.UTC),
+			expected: time.Duration(203) * time.Minute,
+			err:      nil,
+		},
+		{
+			name:     "parseInvalid",
+			duration: "1C",
+			date:     time.Now(),
+			err:      ErrInvalidDuration,
+		},
+		{
+			name:     "parseMissingMultiplier",
+			duration: "DM",
+			date:     time.Now(),
+			err:      ErrInvalidDurationMultiplier,
+		},
```

Comment:
Can we add a test with 0s?
And one with repeating chars?
`"1y4m2y2d1h5m15s"` and `"1y0M2d0m5s"` maybe.

Reply:
should repetition == last one wins, or add them together in your opinion? My initial feeling was the former

Reply:
have pushed a change where last one wins